### ISSUE TITLE
Error loading bundled test .blend files: "BlenderDNA structure encountered an error"

### ISSUE THREAD

**Issue by tellypresence** [assimp/assimp#2920]
When attempting to load bundled test model
```
test/models-nonbsd/BLEND/Bob.blend
```

load fails with error "`BlenderDNA Structure encountered an error`"

Looking at older issues, think this is supposed to have been fixed?  Anyone else getting this?  Thank you

**Comment by SamuelTS** [comment]
From my side, load fails too but with a different error:

`Failed to load file: BLEND: invalid size of file block`

**Comment by YotrolZ** [comment]
@kimkulling Hi, I encountered the same problem, has this problem been fixed?

**Comment by Bronzdragon** [comment]
I've found that, if I change line 419 in BlenderDNA.h (currently `throw DeadlyImportError("Constructing BlenderDNA Structure encountered an error");` to simply `throw;`, blender files import as they did in 4.0.1.

**Comment by turol** [comment]
Might get fixed by https://github.com/assimp/assimp/pull/4679

**Comment by AuracleTech** [comment]
Has been months... is it fixed?
Cause it doesn't look like it, got the issue right now

**Comment by turol** [comment]
`Bob.blend` works for me on latest master. What model are you trying to load?

**Comment by AuracleTech** [comment]
This [pot_light_cam.zip](https://github.com/assimp/assimp/files/10949429/pot_light_cam.zip)
Its a very basic model I made for my game engine, nothing simpler, probably like 60 triangles

**Comment by turol** [comment]
@AuracleTech Does https://github.com/assimp/assimp/pull/5006 fix it?

**Comment by AuracleTech** [comment]
Well, it changed, now I am getting
`Failed to load scene: "BLEND: Expected at least one object with no parent"`

**Comment by turol** [comment]
@AuracleTech Your test model now loads for me after #5006. If you still have problems open a new issue and attach a failing model.

**Comment by stephengold** [comment]
Considering the discussion at https://github.com/assimp/assimp/issues/5348 perhaps this issue should be closed.
